index.html

```diff
@@ -6242,7 +6242,7 @@ <h4 id="processing_of_gyoudori">
           <p its-locale-filter-list="en" lang="en">Set the heading text at the center of the space specified with multiple lines of the kihon-hanmen. For example, when the heading is set at the center of a three line space, it is called  "center of three lines space". Following are some  examples (see [[[#fig3_1_15]]] et al). In these figures, gray rectangles indicate the main text, and dotted rectangles in the heading area indicate the space specified with kihon-hanmen text lines. Also, running headings and page numbers are indicated with gray rectangles. </p>
           <p its-locale-filter-list="ja" lang="ja">基本版面で設定した2行以上の<span class="index" id="d8e1583">複数行の領域の中央に配置する</span>．例えば，基本版面で設定した3行の中央に配置する場合は，“3行取り中央”という．いくつかの配置例を[[[#fig3_1_15]]]以下に示す．なお，以下の図版において，グレーの長方形は本文の文章を示し，見出しのバックにある点線の長方形は基本版面で設定した文字及び行の配置位置を示すものである．</p>
           <aside class="note" id="n212">
-            <p its-locale-filter-list="en" lang="en">The character size of a sub-title is usually two thirds of the main heading character size. The line gap is usually half the heading character size, narrower than the line gaps of main text lines. In the line head alignment case, the sub heading is indented, and in the centering case, the sub heading is also centered. The main heading and the sub heading should not be separated across two pages or two columns. It is expected that the main heading and the sub heading are treated as one object. </p>
+            <p its-locale-filter-list="en" lang="en">The character size of a subtitle is usually two thirds of the main heading character size. The line gap is usually half the heading character size, narrower than the line gaps of main text lines. In the line head alignment case, the sub heading is indented, and in the centering case, the sub heading is also centered. The main heading and the sub heading should not be separated across two pages or two columns. It is expected that the main heading and the sub heading are treated as one object. </p>
```

Comment:
here we seem to use 'subtitle' and 'sub heading' for the same thing.  Note that the image uses 'sub-title', which i'd say is wrong. Note also that it should be  'subheading' rather than 'sub heading'.

Reply:
The image will be fixed after https://github.com/w3c/jlreq/pull/121/files#diff-f42c164a99a6ca4038df5273f1a76fb4 being merged.

Reply:
Yes, this 'sub heading' is ones referred as 'subtitle' and shall be fixed, updated by 06252c4.
I've just regexp-ed for all sub-title other than in [section 4.1.2](https://w3c.github.io/jlreq/#elements-of-block-heading).